public/src/utils.js

```diff
@@ -328,6 +328,10 @@
 			return String(str).replace(new RegExp('<(\\/)?(' + (pattern || '[^\\s>]+') + ')(\\s+[^<>]*?)?\\s*(\\/)?>', 'gi'), '');
 		},
 
+		replaceImgWithAltText: function (str) {
+			return String(str).replace(/<img .*?alt="(.*?)"[^>]*>/gi, '$1');
+		},
+
```

Comment:
`utils.js` is for methods used in both client-side and server-side. Any particular reason the method is added here?

Maybe easier to just define an internal function in `src/topics/teaser.js` since it is only used there, currently.

Reply:
Nope, only because it sounded like an utility function.
Moving into src/topics/teaser.js.

Reply:
Understood. It does, and would fit there, but for now since it is only used in one place, let's keep it simpler. It can be broken out in the future if necessary.